Cargo.toml

```diff
@@ -32,13 +33,17 @@ web-sys = { version = "0.3", features = [
 	"Performance",
 	"Window",
 ] }
+serde_json = "1.0"
```

Comment:
```suggestion
```
Not needed if we guard tests behind `feature = "serde"`.

Reply:
Why is it not needed? Shouldn't it still be imported to test (de)serialization in integration tests?

Reply:
Not sure what I meant either when I said!
Apologies, it was probably late and I was tired :sweat_smile:.